.github/workflows/release.yml

```diff
@@ -21,6 +21,23 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v4
 
+      - name: Replace stable version with the release version
+        run: |-
+          # store the release tag
+          tag="${{ github.ref_name }}"
+
+          # files to search for
+          md_rst_files=$(find . -type f \( -name "*.md" -o -name "*.rst" \))
+
+          # regex patterns
+          badge_pattern="s#(readthedocs\.org/projects/glass/badge/\?version=)stable#\1${tag}#g"
+          url_pattern="s#(glass\.readthedocs\.io.*?)/stable#\1/${tag}#g"
+
+          # perform the replacements
+          for file in $md_rst_files; do
+            sed -E -i "$badge_pattern; $url_pattern" "$file"
+          done
+
```

Comment:
We only need to patch README.md really, since nothing else appears in the sdist or wheels.

Reply:
Okay, I'll change it to that